draft-ietf-ppm-dap.md

```diff
@@ -1812,16 +1812,16 @@ the input shares and defeat privacy.
 This specification defines the following protocol messages, along with their
 corresponding media types types:
 
-- HpkeConfig {{hpke-config}}: "application/ppm-hpke-config"
-- Report {{upload-request}}: "message/ppm-report"
-- AggregateInitReq {{collect-flow}}: "message/ppm-aggregate-init-req"
-- AggregateInitResp {{collect-flow}}: "message/ppm-aggregate-init-resp"
-- AggregateContinueReq {{collect-flow}}: "message/ppm-aggregate-continue-req"
-- AggregateContinueResp {{collect-flow}}: "message/ppm-aggregate-continue-resp"
-- AggregateShareReq {{collect-flow}}: "message/ppm-aggregate-share-req"
-- AggregateShareResp {{collect-flow}}: "message/ppm-aggregate-share-resp"
-- CollectReq {{collect-flow}}: "message/ppm-collect-req"
-- CollectResp {{collect-flow}}: "message/ppm-collect-resp"
+- HpkeConfig {{hpke-config}}: "application/dap-hpke-config"
+- Report {{upload-request}}: "message/dap-report"
+- AggregateInitReq {{collect-flow}}: "message/dap-aggregate-init-req"
```

Comment:
I assume this will conflict now that it's AggregateInitializeReq/Resp?

Reply:
Fixed after rebase.